content/v2.0/influxdb-templates/cloud.md

```diff
@@ -15,12 +15,12 @@ v2.0/tags: [templates]
 products: [cloud]
 ---
 
-If using InfluxDB Cloud, download and use the [`influx` command line interface (CLI)](/v2.0/reference/cli/influx/)
+Use the [`influx` command line interface (CLI)](/v2.0/reference/cli/influx/)
 to apply and manage templates in your InfluxDB Cloud account.
 
-InfluxDB OSS 2.0 includes the InfluxDB CLI (`influx`). If you haven’t already, do the following:
+If you haven’t already, do the following:
 
-1. [Download and install InfluxDB 2.0 OSS](/v2.0/get-started/#start-with-influxdb-oss).
+1. [Download and install `influx` CLI](/v2.0/get-started/#optional-download-and-install-the-influx-cli).
 2. [Configure the `influx` CLI](/v2.0/get-started/#set-up-influxdb) to use your
```

Comment:
does step 2 change with the separate influx install? When I look at this procedure, it seems more for InfluxDB setup than influx CLI setup

Reply:
Yes, definitely. You nailed it here.

Reply:
@sanderson does that mean that step 2 changes?....as is, I left it alone, but when I looked at the procedure referred to in step 2, I couldn't ascertain which bits were meant for the CLI only

Reply:
Sorry, misread your first question. No, step 2 does not need to change.